src/opera/test/data/test_rtc_s1_static_config.yaml

```diff
@@ -0,0 +1,104 @@
+
+            PrimaryExecutable:
+                ProductIdentifier: RTC_S1
+                ProductVersion: "1.0"
+                ProgramPath: mkdir
+                ProgramOptions:
+                    - '-p rtc_s1_test/output_dir/t069_147170_iw1/;'
+                    - 'python3 -c "from opera.util.metadata_utils import create_test_rtc_metadata_product; create_test_rtc_metadata_product(\"rtc_s1_test/output_dir/t069_147170_iw1/rtc_product_v1.0.h5\")";'
+                    - 'dd if=/dev/urandom of=rtc_s1_test/output_dir/t069_147170_iw1/OPERA_L2_RTC-S1-STATIC_T069-147170-IW1_20140403_20230911T203246Z_S1B_30_v1.0_incidence_angle.tif bs=1M count=1;'
+                    - 'dd if=/dev/urandom of=rtc_s1_test/output_dir/t069_147170_iw1/OPERA_L2_RTC-S1-STATIC_T069-147170-IW1_20140403_20230911T203246Z_S1B_30_v1.0_local_incidence_angle.tif bs=1M count=1;'
+                    - 'dd if=/dev/urandom of=rtc_s1_test/output_dir/t069_147170_iw1/OPERA_L2_RTC-S1-STATIC_T069-147170-IW1_20140403_20230911T203246Z_S1B_30_v1.0_number_of_looks.tif bs=1M count=1;'
+                    - 'dd if=/dev/urandom of=rtc_s1_test/output_dir/t069_147170_iw1/OPERA_L2_RTC-S1-STATIC_T069-147170-IW1_20140403_20230911T203246Z_S1B_30_v1.0_rtc_anf_gamma0_to_beta0.tif bs=1M count=1;'
+                    - 'dd if=/dev/urandom of=rtc_s1_test/output_dir/t069_147170_iw1/OPERA_L2_RTC-S1-STATIC_T069-147170-IW1_20140403_20230911T203246Z_S1B_30_v1.0_rtc_anf_gamma0_to_sigma0.tif bs=1M count=1;'
+                    - 'dd if=/dev/urandom of=rtc_s1_test/output_dir/t069_147170_iw1/OPERA_L2_RTC-S1-STATIC_T069-147170-IW1_20140403_20230911T203246Z_S1B_30_v1.0_mask.tif bs=1M count=1;'
+                    - 'dd if=/dev/urandom of=rtc_s1_test/output_dir/t069_147170_iw1/OPERA_L2_RTC-S1-STATIC_T069-147170-IW1_20140403_20230911T203246Z_S1B_30_v1.0.png bs=1M count=1;'
+                    - '/bin/echo RTC-S1 invoked with RunConfig'
+                ErrorCodeBase: 300000
+                SchemaPath: pge/rtc_s1/schema/rtc_s1_sas_schema.yaml
+                IsoTemplatePath: pge/rtc_s1/templates/OPERA_ISO_metadata_L2_RTC_S1_template.xml.jinja2
+                DataValidityStartDate: 20000101
+
+            QAExecutable:
+                Enabled: False
+                ProgramPath:
+                ProgramOptions: []
+
+            DebugLevelGroup:
+                DebugSwitch: False
+                ExecuteViaShell: True  # Must be set to True for test to work
+
+        SAS:
+            runconfig:
+                name: rtc_s1_workflow_default
+
+                groups:
+                    pge_name_group:
+                        pge_name: RTC_S1_PGE
+
+                    input_file_group:
+                        safe_file_path:
+                            - rtc_s1_test/input_dir/SAFE.zip
+                        orbit_file_path:
+                            - rtc_s1_test/input_dir/ORBIT.EOF
+                        burst_id:
+                            - t069_147170_iw1
+
```

Comment:
Do you need  'source_data_access' after 'burst_id', or is it something that we don't need to test?